lib/config.dart

```diff
@@ -307,7 +307,14 @@ class Config {
   /// A list of [CustomToolbar] objects or [DefaultToolbars] constants that define a
   /// custom toolbar.
   ///
-  /// If used the default toolbar no longer shows. Defualts to empty.
+  /// ```dart
+  /// var customToolItem = new CustomToolbar('add_page', 'Add Page', [], 'ic_add_blank_page_white');
```

Comment:
this line looks strange, is that tested?

Reply:
yeah, it works as intended. Kevin reused the CustomToolbar class to also be used for creating custom toolbar items instead of creating a new class

Reply:
that's bad practice, at the minimum should extend to a new class

Reply:
i'll change it to use a different class